.spectral.json

```diff
@@ -1646,6 +1646,17 @@
         "field": "headers"
       },
       "description": "Successful post endpoint responses must have the following \"headers\": \"Location\"."
+    },
+    "required-path-sdk-operation": {
+      "given": [
+        "$.paths[*][get,put,post,delete,options,head,patch,trace]"
+      ],
+      "severity": "error",
+      "then": {
+        "function": "truthy",
+        "field": "x-sdk-operation"
+      },
+      "message": "All operations must include an \"x-sdk-operation\" vendor extension for use in human-readable SDK method names."
```

Comment:
New rule that checks for existence of the `x-sdk-operation` attributes under all path operations.